compiler/ballerina-lang/src/main/resources/compiler.properties

```diff
@@ -775,6 +775,22 @@ error.invalid.listener.var=\
 error.invalid.listener.attachment=\
   invalid listener attachment
 
+error.service.absolute.path.or.literal.required.by.listener=\
+  service absolute path or literal is required by listener
+
+error.service.path.literal.is.not.supported.by.listener=\
+  service path literal is not supported by listener
+
+error.service.absolute.path.is.not.supported.by.listener=\
+  service absolute path is not supported by listener
```

Comment:
Will these be supported in the future?

Reply:
No this is the expected behavior,

if listener attach method is `attach(service object s, string? path)`, then it's an error to have following:
```
service \abs\path on new http:Listener(4000) {

}
```

It should be  either `service "name-literal" on lsn { }` or `service on lsn { }`